src/core/adapters/configuration.ts

```diff
@@ -28,6 +29,9 @@ namespace Adapter {
       }
     });
 
+  export const extractArea = (config: FluxCapacitor.Configuration, defaultValue: string) =>
```

Comment:
if you add a test for this guy it will probably push your coverage % over the one desired by bithound